client/src/dateUtils.js

```diff
@@ -27,30 +27,33 @@ export function dateRangeEndKey(queryKey) {
 }
 
 export function dateToQuery(queryKey, value) {
-  let startKey = dateRangeStartKey(queryKey)
-  let endKey = dateRangeEndKey(queryKey)
+  const startKey = dateRangeStartKey(queryKey)
+  const endKey = dateRangeEndKey(queryKey)
+  const startDate = value.start && moment(null).startOf("day")
+  const startDateEnd = value.start && moment(value.start).endOf("day")
```

Comment:
I would also call the rest startDateStart and endDateEnd, just for better understanding.